csrankings-p.csv

```diff
@@ -349,8 +349,7 @@ Paul R. Tennent,University of Nottingham,https://www.nottingham.ac.uk/computersc
 Paul Ralph,Dalhousie University,http://paulralph.name,oRBuFa0AAAAJ
 Paul Rayson,Lancaster University,https://www.lancaster.ac.uk/scc/about-us/people/paul-rayson,DACFB-kAAAAJ
 Paul Richmond,University of Sheffield,https://www.sheffield.ac.uk/dcs/people/academic/paul-richmond,fLS64xsAAAAJ
-Paul Rosen,University of South Florida,http://www.cspaul.com/wordpress,6iv6AEcAAAAJ
-Paul Rosen 0001,University of South Florida,http://www.cspaul.com/wordpress,6iv6AEcAAAAJ
+Paul Rosen,University of Utah,http://www.cspaul.com/wordpress,6iv6AEcAAAAJ

```

Comment:
I see the alias has changed. You need all of the suffixed ones *as listed in DBLP*, per the instructions.

> Paul Rosen 0001 
> Paul A. Rosen 0001
>